lib/formtools/plugins/plugins-helpers.js

```diff
@@ -226,6 +226,41 @@ var overviewDefaults = function (overview, labels) {
 
     overview.summary.label = overview.summary.label || 'Summary';
 
+
+    //overview Details
+    if (!overview.details || typeof overview.details !== 'object') {
+        // defaults to {}
+        overview.details = {};
+    }
+
+    overview.details.form = overviewDetailsDefault(overview.details.fields, schema, form, labels);
+
+    //set tabs to tabsArray property of overview.details
+    var tabsArray = [];
+    if (Array.isArray(overview.details.tabs) && overview.details.tabs.length) {
+
+        for (var i = 0; i < overview.details.tabs.length; i++) {
```

Comment:
done. Code to process overview.details has been removed from plugins-helpers.js. It has been moved to formtools.overview.